packages/stream-tile-handler/src/tile-document-handler.ts

```diff
@@ -261,7 +265,8 @@ export class TileDocumentHandler implements StreamHandler<TileDocument> {
 
     if (
```

Comment:
this conditional is getting kind of complex to read - can we make a helper function to increase clarity?

Reply:
@stbrody 2 cents from me. It is complex, and it is ok _for now_. In my opinion, this thing should be simplified after we got additional input from external users/partners, not earlier. IMO, it is still not over the edge, crying "simplify me now, or I will bury you".

Reply:
I was thinking about this as well. The most confusing part to me is that it's negatives + ANDs. Don't feel strongly right now, in the future it would definitely make sense to simplify.